source/conf.py

```diff
@@ -199,7 +197,7 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ["_static"]
+html_static_path = ["_static", "_static_generated"]
```

Comment:
This would be undone with the long comment above.

Reply:
concur.

Do we care much to differentiate a `_static` folder which has contents which ought to be in version control and edited, and between a `_static` folder which is generated at build time? That's the only reason I tacked on the suffix.

Reply:
RE: CSS file - I haven't yet moved it yet for this answer.... any thoughts on where it (and the generated js files) ought to live?

Reply:
The _static folder doesn't matter. Any extension can dump whatever it wants in there.

The css file should go somewhere in the extension folder.
Either, you can have a pre-populated _static folder that already contains the css file and generate the js file into the same folder.
Or, you can have a _static and _static_generated folder and add them both in the app.connect.

Reply:
Got it. I think this is finalized now. Closing this out, but feel free to re-open if I ended up off in the weeds.